test/libsolidity/syntaxTests/memberLookup/memory_structs_with_mapping_array_struct_array.sol

```diff
@@ -9,4 +9,4 @@ contract Test {
     }
 }
 // ----
-// TypeError: (208-218): Member "b1" is not available in struct Test.S1 memory outside of storage.
```

Comment:
I think this error message and the corresponding helper function can now be deleted, can it?